release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1930,9 +1932,11 @@ This behavior has been fixed, and the references are now accepted and correctly
 
 * The `vsphereStorageDriver` field of the Storage Operator object has been deprecated and will be removed in a future release. This field was used to opt in to CSI migration on {product-title} 4.13 vSphere clusters, but it has no effect on {product-title} 4.14 and newer clusters. (link:https://issues.redhat.com/browse/OCPBUGS-13914[*OCPBUGS-13914*])
 
-[discrete]
-[id="ocp-4-14-windows-containers-bug-fixes"]
-==== Windows containers
+//[discrete]
+//[id="ocp-4-14-windows-containers-bug-fixes"]
+//==== Windows containers
+// Added after OCP GA
+//* Previously, Window nodes could not be deconfigured because of conatainerd log files that were not removed. Now, containered runtime is stopped before removing the log files, fixing the issue. (link:https://issues.redhat.com/browse/OCPBUGS-14700[*OCPBUGS-14700*])
```

Comment:
```suggestion
//* Previously, Window nodes could not be deconfigured because of `containerd` log files that were not removed. Now, `containerd` runtime is stopped before removing the log files, fixing the issue. (link:https://issues.redhat.com/browse/OCPBUGS-14700[*OCPBUGS-14700*])
```